app/controllers/assessment/autograde.rb

```diff
@@ -10,8 +10,9 @@ module AssessmentAutograde
   # method called when Tango returns the output
   # action_no_auth :autograde_done
   def autograde_done
-    @course = Course.find(params[:course_id]) || (render(nothing: true) && return)
-    @assessment = @course.assessments.find(params[:id])
+
+    @course = Course.find_by_name(params[:course_id]) || (render(nothing: true) && return)
```

Comment:
Are we changing the callback url as well?  If we didn't this actually doesn't need to be changed.